Core.Collectors/IO/AzureBlobRecordWriter.cs

```diff
@@ -31,20 +31,20 @@ public class AzureBlobRecordWriter<T> : RecordWriterCore<T> where T : FunctionCo
                                      ITelemetryClient telemetryClient,
                                      T functionContext,
                                      ContextWriter<T> contextWriter,
-                                     string storageConnectionEnvironmentVariable = "AzureWebJobsStorage",
+                                     string storageAccountEnvironmentVariable = "StorageAccountName",
```

Comment:
Careful about this change especially for GitHubcollectors. AzureWebJobsStorage has a "special" meaning referring to the storage account where the Azure Functions are running. This is also what we use to output the data in GH collectors (compared to what we do with ADO collectors where the data storage account is different).